variables.tf

```diff
@@ -20,4 +20,12 @@ variable "GIT_HUB_SECRET"{
 
 variable "LOGIN_DISABLED"{
     default = "false"
+}
+
+variable "LOG_LEVEL"{
+    default = "DEBUG"
```

Comment:
Worth considering whether you want to default to DEBUG in production - it does give you the greatest level of detail, but also the greatest level of noise. Typically I'd expect to aim for INFO, but it does depend on what you're tracking, and how your team chooses to use each level of logging